packages/app-utils/src/storage/grpc/GrpcFileStorageTable.test.ts

```diff
@@ -1,20 +1,22 @@
 import { FileUtils } from '@deephaven/file-explorer';
-import type { ItemDetails, StorageService } from '@deephaven/jsapi-types';
+import type { dh } from '@deephaven/jsapi-types';
 import GrpcFileStorageTable from './GrpcFileStorageTable';
 
-let storageService: StorageService;
+let storageService: dh.storage.StorageService;
 function makeTable(baseRoot = '', root = ''): GrpcFileStorageTable {
   return new GrpcFileStorageTable(storageService, baseRoot, root);
 }
 
-function makeStorageService(): StorageService {
+function makeStorageService(): dh.storage.StorageService {
   return {
     listItems: jest.fn(async () => []),
     loadFile: jest.fn(async () => {
       throw new Error('No file loaded');
     }),
     deleteItem: jest.fn(async () => undefined),
-    saveFile: jest.fn(async () => undefined),
+    saveFile: jest.fn(
```

Comment:
This might be a good spot for 

```typescript
TestUtils.createMockProxy<dh.storage.StorageService>({
  loadFile: jest.fn(async () => {
      throw new Error('No file loaded');
  }),
})
```